### PR TITLE
Nova 4 Upgrade

### PR DESCRIPTION
**Pull request by ali-raza-saleem**

### CODE REVIEW

**Comment by remkobrenters** [comment]
Mr @RVxLab do you want to look into this PR?

**Comment by RVxLab** [comment]
I can't make a proper judgement call on this as I don't use Nova 4. The only suggestion I have is to set the PHP requirement to `^7.3|^8` as 7.3 is the minimum now.

**Comment by remkobrenters** [comment]
That's fast :) thanks. Makes sense. @dgerrese do you think you can look into this as I have no Nova 4 running anywhere yet.

File: composer.json [hunk 1]
```diff
@@ -8,13 +8,19 @@
     "license": "MIT",
     "require": {
         "php": ">=7.1.0",
```
Comment (RVxLab) [inline composer.json]:
```suggestion
        "php": "^7.3|^8",
```

**Review by RVxLab — CHANGES_REQUESTED** [review]
I looked a little more in depth, nothing looks particularly wrong but someone else needs to do a final call.

Also @the-alichemist if you can please minify the JS that'd be great.

**Comment by dgerrese** [comment]
Tested this in Nova 4 and seems to work.
I don't see any issues other than @RVxLab's suggestions.

**Comment by remkobrenters** [comment]
@the-alichemist are you willing to change the PR a bit so we can merge it?

**Comment by ali-raza-saleem** [comment]
@RVxLab @remkobrenters Sorry for the late reply. Suggestions have been implemented.

**Review by RVxLab — APPROVED** [review]

**Comment by RVxLab** [comment]
Thank you for the PR!

**Pull request merged**